setup.py

```diff
@@ -14,6 +14,7 @@
     'requests-toolbelt>=0.7.0',
```

Comment:
Seems like latest `requests-toolbelt` broke CI in the latest release: https://github.com/requests/toolbelt/issues/241